tempodb/encoding/iterator_paged.go

```diff
@@ -4,11 +4,13 @@ import (
 	"context"
 	"fmt"
 	"io"
-	"math"
 
 	"github.com/grafana/tempo/tempodb/encoding/common"
 )
 
+const intSize = 32 << (^uint(0) >> 63) // 32 or 64
```

Comment:
This is pulled out of the math package. We had to do this because Google Cloud Functions supports 1.16 as the latest runtime and these constants were defined after that.

Reply:
Can we add a comment here with something like "use math.MaxInt once Google Cloud Functions supports Go 1.17"?

Reply:
- Changed go.mod to be 1.16
- Added requested comment
- Also had to drop use of time.UnixMicro()